README.md

```diff
@@ -36,20 +36,24 @@ A multi-language, cross-platform library that enables different simulators to ea
 
 ## Introduction
 
-Welcome to the repository for the Hierarchical Engine for Large-scale Infrastructure Co-Simulation (HELICS). HELICS provides a general-purpose, modular, highly-scalable co-simulation framework that runs cross-platform and has bindings for multiple languages. It is a library that enables multiple existing simulation tools (and/or instances of the same tool), known as "federates", to exchange data during runtime and stay synchronized in time such that together they act as one large simulation, or "federation". This enables bringing together simulation tools from multiple domains to form a complex software-simulation without having to change the individual tools.
+Welcome to the repository for the Hierarchical Engine for Large-scale Infrastructure Co-Simulation (HELICS). HELICS provides a general-purpose, modular, highly-scalable co-simulation framework that runs cross-platform and has bindings for multiple languages. It is a library that enables multiple existing simulation tools (and/or instances of the same tool), known as "federates", to exchange data during runtime and stay synchronized in time such that together they act as one large simulation, or "federation". This enables bringing together simulation tools from multiple domains to form a complex software simulation without having to change the individual tools.
```

Comment:
I feel like this first paragraph is still a bit complex, and the latter part is saying roughly the same thing as the new paragraph (but it in a more complex way) -- maybe it can be simplified a bit more, and possibly move the new paragraph to right below it?

Reply:
Hmm, I actually only removed an extraneous hyphen otherwise unchanged. Tempted to leave as is (with hyphen fix) for now

Reply:
I do like the idea of swapping paragraphs and have made that change in a new commit to this PR. Thanks for the suggestion